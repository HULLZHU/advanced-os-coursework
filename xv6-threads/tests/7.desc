testing thread_create() and thread_join() system call with a simple thread function.