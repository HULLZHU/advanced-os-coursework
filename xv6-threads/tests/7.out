XV6_TEST_OUTPUT : i before threadfunc = 0
XV6_TEST_OUTPUT : Inside threadfunc
XV6_TEST_OUTPUT : Lock Acquired inside threadfunc
XV6_TEST_OUTPUT : Lock Released inside threadfunc
XV6_TEST_OUTPUT : i after threadfunc = 31415
